app/models/sample.rb

```diff
@@ -46,7 +46,7 @@ def initiate_s3_cp
     fastq2 = input_files[1].source
     command = "aws s3 cp #{fastq1} #{sample_input_s3_path}/;"
     command += "aws s3 cp #{fastq2} #{sample_input_s3_path}/;"
-    if s3_preload_result_path.present?
+    if s3_preload_result_path[0..4] == 's3://'
```

Comment:
Don't you need to keep the presence check, before the prefix check?   In my samples db (fresh import from DEV), there are lots of legit-looking rows with NULL s3_preload_result_path, and in the rails console calling [0..4] on a nil value raises an exception.